strawberryfields/gbs/plot.py

```diff
@@ -252,11 +259,94 @@ def plot_subgraph(subgraph: nx.Graph, size: int = 500) -> None:  # pragma: no co
         xaxis=dict(showgrid=False, zeroline=False, showticklabels=False),
         yaxis=dict(showgrid=False, zeroline=False, showticklabels=False),
         margin=dict(b=0, l=0, r=0, t=25),
-        height=size,
-        width=size,
+        height=plot_size,
+        width=plot_size,
         plot_bgcolor="#ffffff",
     )
 
     f = go.Figure(data=[g_edges, g_nodes], layout=layout)
 
     return f
+
+
+def plot_points(R: np.ndarray, sample: Optional[list] = None,
+                plot_size: int = 500, point_size: int = 30) -> None:  # pragma: no cover
+    """Creates a Plotly plot of two-dimensional points given their input coordinates. Sampled
+    points can be optionally highlighted among all points.
+
+    **Example usage:**
+
+    >>> R = np.random.normal(0, 1, (50, 2))
+    >>> sample = [1] * 10 + [0] * 40  # select first ten points
+    >>> plot_points(R, sample).show()
+
+    .. image:: ../../_static/normal_pp.png
+       :width: 40%
+       :align: center
+       :target: javascript:void(0);
+
+    Args:
+        R (np.array): Coordinate matrix. Rows of this array are the coordinates of the points.
+        sample (list[int]): optional subset of sampled points to be highlighted
+
+    Returns:
+         Figure: Plotly figure of points with the sample highlighted
```

Comment:
No need to repeat plotly here, right?
```suggestion
         Figure: figure of points with the sample highlighted
```

Reply:
Possibly agree, although we should then remove the "Plotly" in the instances above for plotting graphs and subgraphs